tox.ini

```diff
@@ -5,23 +5,23 @@ skipsdist = True
 [testenv:py27-lint]
 commands = bash .ci/flake8_wrapper.sh
 whitelist_externals = bash
-deps = -r.ci/tox_envs/flake8/requirements.txt
+deps = -rlib/galaxy/dependencies/pipfiles/flake8/pinned-requirements.txt
```

Comment:
Why use `pinned-requirements.txt` instead of `pinned-hashed-requirements.txt` (which is what `.ci/tox_envs/flake8/requirements.txt` previously was)?

Reply:
I guess we might as well use the hashed requirements - I was worried it was maybe more brittle (in case the hashes are wrong somehow or the version of pip used is too old and doesn't support them).

I'll switch this.

Reply:
We can expect developers and Travis to have a recent `pip`, I suppose. And `scripts/common_startup.sh` by default installs `pip>=8.1` .